docs/DOCKING_GUIDE.md

```
@@ -280,52 +303,55 @@ SUPABASE_SERVICE_KEY=your-service-key
-# Agent Zero MCP Docking
-AGENT_ZERO_MCP_ENABLED=true
-AGENT_ZERO_MCP_TOKEN=\$(openssl rand -hex 16)
+# Start in docked mode
+make docked
 
-# Neo4j Dual-Write
-NEO4J_LOCAL_PASSWORD=\$(openssl rand -hex 16)
-NEO4J_PARENT_PASSWORD=\${NEO4J_PARENT_PASSWORD}
+# Or manually with override
+docker compose -f docker-compose.yml -f docker-compose.docked.yml \
+  --env-file .env.local up -d --build
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)